sql-delta-import/src/main/scala/com/razorpay/spark/jdbc/JDBCImport.scala

```diff
@@ -150,25 +150,27 @@ class JDBCImport(
   private def readJDBCSourceInParallel(): DataFrame = {
 
     if (importConfig.splitBy.nonEmpty) {
+      val defaultString="defaultValue"
       val (lower, upper) = spark.read
         .jdbc(buildJdbcUrl, importConfig.boundsSql, jdbcParams)
-        .selectExpr("cast(min as long) min", "max")
-        .as[(Option[Long], Option[Long])]
+        .selectExpr("cast(min as string) min", "cast(max as string) max")
+        .as[(Option[String], Option[String])]
         .take(1)
-        .map { case (a, b) => (a.getOrElse(0L), b.getOrElse(0L)) }
+        .map { case (a, b) => (a.getOrElse(defaultString), b.getOrElse(defaultString)) }
         .head
 
-      spark.read
-        .jdbc(
-          buildJdbcUrl,
-          importConfig.jdbcQuery,
-          importConfig.splitColumn,
-          lower,
-          upper,
-          importConfig.chunks,
-          jdbcParams
-        )
-        .where(s"${importConfig.splitColumn} >= $lower and ${importConfig.splitColumn} <= $upper")
+      spark.read.format("jdbc")
+        .option("url",buildJdbcUrl)
+        .option("dbtable",importConfig.jdbcQuery)
+        .option("partitionColumn",importConfig.splitColumn)
+        .option("lowerBound",lower)
```

Comment:
what would happen if the lower or upper is set to default i.e. "defaultValue".
what would be the behaviour here then?

Reply:
It will produce error in that case.

<img width="1344" alt="Screenshot 2022-09-09 at 3 48 51 PM" src="https://user-images.githubusercontent.com/112069583/189328573-34cb3b15-307d-4507-808c-bce5dd622d1e.png">
Screenshot of that error when I tried it on my local setup.